include/sys/socket.h

```diff
@@ -284,23 +284,13 @@
  * aligned at an appropriate boundary so that pointers to it can be cast
  * as pointers to protocol-specific address structures and used to access
  * the fields of those structures without alignment problems.
- *
- * REVISIT: sizeof(struct sockaddr_storge) should be 128 bytes.
  */
 
-#ifdef CONFIG_NET_IPv6
 struct sockaddr_storage
 {
   sa_family_t ss_family;       /* Address family */
-  char        ss_data[26];     /* 26-bytes of address data */
+  char        ss_data[126];    /* 126-bytes of address data */
```

Comment:
`126` is quite expensive. Maybe we can change to 32 to handle `rexecd` and extend in case if it will be needed in future?

Reply:
Since sockaddr_un is 110 bytes:
```
#define UNIX_PATH_MAX  108

struct sockaddr_un
{
  sa_family_t sun_family;        /* AF_UNIX */
  char sun_path[UNIX_PATH_MAX];  /* pathname */
};
```
Per the spec, we have to extend to 110 which is close to 128 anyway.:(

Reply:
I see. Thank you!

Reply:
@pkarashchenko do you finish setup apache account? If so, you could merge the PR after you finish review for the simple change like this:).

Reply:
> @pkarashchenko do you finish setup apache account? If so, you could merge the PR after you finish review for the simple change like this:).

It's on the way :) I'll join team soon.